python/dgl/distributed/graph_partition_book.py

```diff
@@ -9,12 +9,12 @@
 from .._ffi.ndarray import empty_shared_mem
 
 def _move_metadata_to_shared_mam(graph_name, num_nodes, num_edges, part_id,
-                                 num_partitions, node_map, edge_map):
-    ''' Move all metadata to the shared memory.
+                                 num_partitions, node_map, edge_map, is_range_part):
+    ''' Move all metadata of the partition book to the shared memory.
 
     We need these metadata to construct graph partition book.
     '''
-    meta = _to_shared_mem(F.tensor([num_nodes, num_edges,
+    meta = _to_shared_mem(F.tensor([int(is_range_part), num_nodes, num_edges,
```

Comment:
is_range_part is a bool? We need to convert it to int tensor with 0 or 1?
Both torch and tensorflow support bool tensor

Reply:
yes, but the rest of the elements are integer. we have to convert it to integer as well.